docs/pages/GettingStarted/Extended/configure_topologies.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configure yours deployment topologies
+weight: 3
+---
+
+In this tutorial we want to simulate 3 realistic scenarios, considering the cluster-1 as the **cluster home**:
```

Comment:
I would suggest to describe the above 3 scenarios in a more high-level way. The current description is fine, but it's a technical description, not business. We should present them both.
E.g., (for the first case): This simulates a case in which a service has to replicated on multiple clusters, e.g., all the ones that belongs to the same geographical area.

Reply:
yes thanks for the advice, we are just working to improve this aspect.